.github/workflows/develop.yml

```diff
@@ -1,58 +1,31 @@
-name: Build BTS and Run Build and Test in Container
+name: Push nightly BTS Image
```

Comment:
There are two different notations, `Image` and `Images`.
I'd like to see everything standardized as it's different in several places.

Reply:
@mdonaka 
Noted with thanks.  
I checked that there are no differences by `grep image -nri .github/`.